shippo/api_requestor.py

```diff
@@ -90,6 +90,11 @@ def request_raw(self, method, url, params=None):
         Mechanism for issuing an API call
         """
         from shippo.config import api_version
+        from shippo.config import app_name
+        from shippo.config import app_version
+
+        if app_version == '':
+            app_version = VERSION
```

Comment:
nit: if we're not going to default both `app_name` and `app_version` to empty string, we might also default `app_name` to something sensible here.  (Seems odd to default one value but not the other.)

Reply:
We can have the default app_name and version as 1.0  ==>  "PythonApp/1.0'